includes/gateways/paypal/admin/connect.php

```diff
@@ -252,122 +254,132 @@ function get_account_info() {
+			$webhook_message  .= $validator->errors_for_webhook->get_error_message();
+
+			if ( in_array( 'webhook_missing', $validator->errors_for_webhook->get_error_codes() ) ) {
+				$actions['webhook'] = '<button type="button" class="button edd-paypal-connect-action" data-nonce="' . esc_attr( wp_create_nonce( 'edd_create_paypal_webhook' ) ) . '" data-action="edd_paypal_commerce_create_webhook">' . esc_html__( 'Create Webhook', 'easy-digital-downloads' ) . '</button>';
 			}
+		} else {
+			$webhook_dashicon = 'yes';
+			$webhook_message  .= __( 'Webhook successfully configured for the following events:', 'easy-digital-downloads' );
+		}
 
-			$missing_events = array_diff( $expected_events, $actual_events );
-			$number_missing = count( $missing_events );
-			if ( $number_missing ) {
-				$status         = 'warning';
-				$webhook_status = wpautop( _n(
-					'<strong>Warning:</strong> Webhook is configured but is missing an event. Click "Sync Webhook" to correct this.',
-					'<strong>Warning:</strong> Webhook is configured but is missing events. Click "Sync Webhook" to correct this.',
-					$number_missing,
-					'easy-digital-downloads'
-				) );
-			}
+		ob_start();
+		?>
+		<li>
+			<span class="dashicons dashicons-<?php echo esc_attr( $webhook_dashicon ); ?>"></span>
+			<span><?php echo wp_kses( $webhook_message, array( 'strong' => array() ) ); ?></span>
+			<?php if ( $validator->webhook ) : ?>
+				<ul class="edd-paypal-webhook-events">
+					<?php foreach ( array_keys( PayPal\Webhooks\get_webhook_events() ) as $event_name ) : ?>
+						<li>
+							<span class="dashicons dashicons-<?php echo in_array( $event_name, $validator->enabled_webhook_events ) ? 'yes' : 'no'; ?>"></span>
+							<span class="edd-paypal-webhook-event-name"><?php echo esc_html( $event_name ); ?></span>
+						</li>
+					<?php endforeach; ?>
+				</ul>
+			<?php endif; ?>
+		</li>
+		<?php
+		$account_status .= ob_get_clean();
 
-			ob_start();
-			?>
-			<ul class="edd-paypal-webhook-events">
-				<?php foreach ( $expected_events as $event_name ) : ?>
-					<li>
-						<span class="dashicons dashicons-<?php echo in_array( $event_name, $actual_events ) ? 'yes' : 'no'; ?>"></span>
-						<span class="edd-paypal-webhook-event-name"><?php echo esc_html( $event_name ); ?></span>
-					</li>
-				<?php endforeach; ?>
-			</ul>
-			<?php
-			$webhook_status .= ob_get_clean();
-		} catch ( \Exception $e ) {
-			$status         = 'warning';
-			$webhook_status = wpautop( __( '<strong>Warning:</strong> Webhook not configured. Some actions may not be working properly.', 'easy-digital-downloads' ) );
-			$actions        = array(
-				'<button type="button" class="button edd-paypal-connect-action" data-nonce="' . esc_attr( wp_create_nonce( 'edd_create_paypal_webhook' ) ) . '" data-action="edd_paypal_commerce_create_webhook">' . esc_html__( 'Create Webhook', 'easy-digital-downloads' ) . '</button>'
-			);
+		if ( ! edd_is_gateway_active( 'paypal_commerce' ) ) {
```

Comment:
What would you think about moving this to the beginning of the output? I feel like it is a little bit lost down here after the webhooks and I'm likely to miss it. Possibly also add a `dashicons-no` to the beginning, just a gray one--to be consistent with the other output but not present a true warning?

![image](https://user-images.githubusercontent.com/4853066/123653478-2b690c80-d7fb-11eb-8add-127aa0212a10.png)

Reply:
Can you copy this comment to the [main `paypal-commerce` PR](https://github.com/easydigitaldownloads/easy-digital-downloads/pull/8596)? I think that's text from the original PR and I'll put this down as "nice to add, but not high priority right now". (In other words, I'll look into it once the really important stuff is done, which is probably after this particular PR is merged.)